### PR TITLE
Generate ext/sqlite3 class synopses from stubs

### PR DESCRIPTION
**Pull request by kocsismate**

### CODE REVIEW

**Comment by cmb69** [comment]
Thank you, Máté! I'm not sure about https://github.com/php/doc-en/runs/4136624955?check_suite_focus=true#step:6:33. It might be a good idea to get rid of these superfluous error messages first.

**Comment by kocsismate** [comment]
Thanks for pointing this out! I added the 2 undocumented constructors. The documentation is still missing, but at least the methods are there..

File: reference/sqlite3/sqlite3result/construct.xml [hunk 1]
```diff
@@ -0,0 +1,60 @@
+<?xml version="1.0" encoding="utf-8"?>
+<refentry xml:id="sqlite3result.construct" xmlns="http://docbook.org/ns/docbook" xmlns:xlink="http://www.w3.org/1999/xlink">
+ <refnamediv>
+  <refname>SQLite3Result::__construct</refname>
+  <refpurpose>Description</refpurpose>
```
Comment (cmb69) [inline reference/sqlite3/sqlite3result/construct.xml:5]:
```suggestion
  <refpurpose>Constructs a SQLite3Result</refpurpose>
```

File: reference/sqlite3/sqlite3result/construct.xml [hunk 2]
```diff
@@ -0,0 +1,60 @@
...
+ </refnamediv>
+
+ <refsect1 role="description">
+  &reftitle.description;
+  <constructorsynopsis>
+   <modifier>private</modifier> <methodname>SQLite3Result::__construct</methodname>
+   <void />
+  </constructorsynopsis>
+  <para>
+
```
Comment (cmb69) [inline reference/sqlite3/sqlite3result/construct.xml:15]:
```suggestion
   <classname>SQLite3Result</classname> instances are created by
   <methodname>SQLite3::query> and <methodname>SQLite3Stmt::execute</methodname>.
```

File: reference/sqlite3/sqlite3result/construct.xml [hunk 3]
```diff
@@ -0,0 +1,60 @@
...
+  </para>
+
+  &warn.undocumented.func;
+
```
Comment (cmb69) [inline reference/sqlite3/sqlite3result/construct.xml:19]:
```suggestion
```

File: reference/sqlite3/sqlite3result/construct.xml [hunk 4]
```diff
@@ -0,0 +1,60 @@
...
+ <refsect1 role="returnvalues">
+  &reftitle.returnvalues;
+  <para>
+   
+  </para>
+ </refsect1>
+ -->
+
+
+
```
Comment (cmb69) [inline reference/sqlite3/sqlite3result/construct.xml:39]:
```suggestion
```

File: reference/sqlite3/sqlite3stmt/construct.xml [hunk 5]
```diff
@@ -0,0 +1,78 @@
+<?xml version="1.0" encoding="utf-8"?>
+<refentry xml:id="sqlite3stmt.construct" xmlns="http://docbook.org/ns/docbook" xmlns:xlink="http://www.w3.org/1999/xlink">
+ <refnamediv>
+  <refname>SQLite3Stmt::__construct</refname>
+  <refpurpose>Description</refpurpose>
```
Comment (cmb69) [inline reference/sqlite3/sqlite3stmt/construct.xml:5]:
```suggestion
  <refpurpose>Constructs a SQLite3Stmt</refpurpose>
```

File: reference/sqlite3/sqlite3stmt/construct.xml [hunk 6]
```diff
@@ -0,0 +1,78 @@
...
+
+ <refsect1 role="description">
+  &reftitle.description;
+  <constructorsynopsis>
+   <modifier>private</modifier> <methodname>SQLite3Stmt::__construct</methodname>
+   <methodparam><type>SQLite3</type><parameter>sqlite3</parameter></methodparam>
+   <methodparam><type>string</type><parameter>query</parameter></methodparam>
+  </constructorsynopsis>
+  <para>
+
```
Comment (cmb69) [inline reference/sqlite3/sqlite3stmt/construct.xml:16]:
```suggestion
   <classname>SQLite3Stmt</classname> instances are created by
   <methodname>SQLite3::prepare</methodname>.
```

File: reference/sqlite3/sqlite3stmt/construct.xml [hunk 7]
```diff
@@ -0,0 +1,78 @@
...
+  </para>
+
+  &warn.undocumented.func;
+
```
Comment (cmb69) [inline reference/sqlite3/sqlite3stmt/construct.xml:20]:
```suggestion
```

File: reference/sqlite3/sqlite3stmt/construct.xml [hunk 8]
```diff
@@ -0,0 +1,78 @@
...
+ <refsect1 role="returnvalues">
+  &reftitle.returnvalues;
+  <para>
+   
+  </para>
+ </refsect1>
+ -->
+
+
+
```
Comment (cmb69) [inline reference/sqlite3/sqlite3stmt/construct.xml:57]:
```suggestion
```

**Review by cmb69 — COMMENTED** [review]
Ah, there are actually private constructors! I think we should finish up there documentation. :)

File: reference/sqlite3/sqlite3result/construct.xml [hunk 9]
```diff
@@ -0,0 +1,45 @@
+<?xml version="1.0" encoding="utf-8"?>
+<refentry xml:id="sqlite3result.construct" xmlns="http://docbook.org/ns/docbook" xmlns:xlink="http://www.w3.org/1999/xlink">
+ <refnamediv>
+  <refname>SQLite3Result::__construct</refname>
+  <refpurpose>Constructs a SQLite3Result</refpurpose>
```
Comment (kocsismate) [inline reference/sqlite3/sqlite3result/construct.xml:5]: Shouldn't it be `an SQLite3Result`? ^^ At least I pronounce it like that (and I found an example for the "an" article)
```suggestion
  <refpurpose>Constructs a SQLite3Result</refpurpose>
```

Comment (cmb69) [reply reference/sqlite3/sqlite3result/construct.xml:5]: Yes, that is a tricky one. Many pronounce it "sequel", so I'm not sure.

Comment (kocsismate) [reply reference/sqlite3/sqlite3result/construct.xml:5]: Apparently, the docs settled on using `an Sqlite3` so far, so I'd continue using it.

**Review by cmb69 — APPROVED** [review]
Thank you!

**Pull request merged**